decidim-core/app/events/decidim/report_created_event.rb

```diff
@@ -0,0 +1,28 @@
+# frozen-string_literal: true
+
+module Decidim
+  class ReportCreatedEvent < Decidim::Events::SimpleEvent
+    delegate :url_helpers, to: "Decidim::Core::Engine.routes"
+
+    i18n_attributes :resource_path, :report_reason, :resource_title
+
+    def resource_path
+      # TODO: use url_helpers to get the path to the reported resource
+      "/reported_resource"
+    end
+
+    def resource_url
+      # TODO: use url_helpers to get the url to the reported resource
+      "localhost:3000/reported_resource"
+    end
+
+    def resource_title
```

Comment:
The default `resource_title` might be enough, it tries to get a meaningful title: https://github.com/decidim/decidim/blob/develop/decidim-core/lib/decidim/events/base_event.rb#L72-L82